Expensify/App
# Added tapable links to user's login information on DetailsPage

### Details


### Fixed Issues

$ Fixes #3476

### Tests | QA Steps
1. Open Any conversation on E.cash.
2. Click on the other user's name on the Chat Title.
3. Details page should open.
4. Now Click on Either the mobile number of the user or its email.
5. Clicking on the mobile Number should open the Dialer.
6. Clicking on Email should prompt for mailing.

### Tested On


### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/124740721-ac7d7d80-df38-11eb-99b1-17f055496153.mp4


#### Mobile Web


https://user-images.githubusercontent.com/24370807/124740743-b1423180-df38-11eb-8cbb-22c7334cb03b.mp4

#### iOS


#### Android


https://user-images.githubusercontent.com/24370807/124741315-2f063d00-df39-11eb-81de-8db1ff539755.mp4
